src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -234,6 +236,7 @@
   <Target Name="CreateVersionFileDuringBuild" Condition="'$(SkipVersionGeneration)'!='true' AND '$(ShouldCreateVersionFileDuringBuild)'=='true'" DependsOnTargets="CreateOrUpdateCurrentVersionFile">
     <PropertyGroup>
       <BuildNumberMajor>$(GeneratedBuildNumberMajor)</BuildNumberMajor>
```

Comment:
We will need to think through this some more but I was kind of hoping we could check-in something for locking the versions in servicing and not require the correct things be passed in to the build.